pyproject.toml

```diff
@@ -1,7 +1,52 @@
+[project]
+name = "akkudoktor-eos"
+version = "0.0.1"
+authors = [
+  { name="Andreas Schmitz", email="author@example.com" },
+]
+description = "This project provides a comprehensive solution for simulating and optimizing an energy system based on renewable energy sources. With a focus on photovoltaic (PV) systems, battery storage (batteries), load management (consumer requirements), heat pumps, electric vehicles, and consideration of electricity price data, this system enables forecasting and optimization of energy flow and costs over a specified period."
+readme = "README.md"
+license = {file = "LICENSE"}
+requires-python = ">=3.8"
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Programming Language :: Python :: 3",
+    "Operating System :: OS Independent",
+]
+dependencies = [
+    "numpy>=1.26.4",
+    "matplotlib>=3.9.2",
+    "flask>=3.0.3",
+    "scikit-learn>=1.5.2",
+    "deap>=1.4.1",
+    "joblib>=1.4.2",
+    "requests>=2.32.3",
+    "pytest>=8.3.3",
+    "pytest-cov>=5.0.0",
+    "pandas>=2.2.3",
+]
```

Comment:
Dependencies should remain in `requirements.txt`.
I am not familiar with hatchling, but when using setuptools as the build system it works like this:

https://github.com/JuBiotech/peak-performance/blob/efb527b6d811a760d55a3a6db83b0dbae13488d8/pyproject.toml#L25-L27

Reply:
I will give setuptools a try.

Reply:
Switched to setuptools.